Assets/com.fluid.behavior-tree/Runtime/Tasks/TaskStatus.cs

```diff
@@ -1,7 +1,7 @@
 ﻿namespace CleverCrow.Fluid.BTs.Tasks {
-    public enum TaskStatus {
-        Success,
-        Failure,
-        Continue
+    public enum TaskStatus:byte {
```

Comment:
I love this idea, but for various architectural and serialization reasons, I want to leave this a vanilla enum. It will probably break serialization on existing projects too. Potentially even crash some 3rd party GUI libraries and other problems.